docs/tutorials/google/start.ipynb

```diff
@@ -504,8 +505,8 @@
         "        gate_set=cirq.google.SYC_GATESET)\n",
         "\n",
         "    print(\"Scheduled. View the job at: https://console.cloud.google.com/quantum/\"\n",
-        "          \"programs{}?mods=quantum_ng2&project={}\".format(\n",
-        "              job.job_resource_name.split('/programs')[1], project_id))\n",
+        "         f\"programs/{job.program_id}/jobs/{job.job_id}\"\n",
+        "         f\"/overview?project={project_id}\")\n",
```

Comment:
I think we should put the URL-construction into an easily discoverable function somewhere. WDYT?

(Out of scope for this PR.)

Reply:
Yeah, that's a good idea.  I'll add it to my TODOs.

Reply:
Filed #3305.